etc/org.opencastproject.transcription.ibmwatson.IBMWatsonTranscriptionService.cfg

```diff
@@ -1,37 +1,49 @@
 # Change enabled to true to enable this service.
 enabled=false
 
-# User obtained when registering with the IBM Watson Speech-to_text service
-ibm.watson.user=<SERVICE_USER>
+# IBM Watson Speech-to-Text service url
+# Default: https://stream.watsonplatform.net/speech-to-text/api
+# ibm.watson.service.url=https://stream.watsonplatform.net/speech-to-text/api
+
+# APi key obtained when registering with the IBM Watson Speech-to_text service.
+# If empty, user and password below will be used.
+ibm.watson.api.key=<API_KEY>
```

Comment:
Shouldn't this be commented out by default?

Reply:
Yes, will do!